src/_scss/pages/awardV2/idv/_awardFundingSummary.scss

```diff
@@ -101,6 +101,13 @@
       }
     }
   }
+  .federal-accounts__section--note {
+    font-size: 14px;
```

Comment:
@MaxwellKendall can you add `width: 100%;` please :)

Reply:
Yes sir